packages/web/src/common/store/tipping/selectors.ts

```diff
@@ -173,3 +165,122 @@ export const getOptimisticSupporters = createSelector(
     return result
   }
 )
+
+export const getOptimisticSupportingForUser = (
+  state: CommonState,
+  userId: ID
+) => getOptimisticSupporting(state)[userId]
+
+export const getOptimisticSupportersForUser = (
+  state: CommonState,
+  userId: ID
+) => getOptimisticSupporters(state)[userId]
+
+export const getOptimisticUserIdsIfNeeded = (
```

Comment:
Should probably also rewrite this to leverage reselect in the future - let's make a linear for that

Reply:
added to PR so no need for linear. thx